docs/eventing/broker/kafka-broker.md

```diff
@@ -0,0 +1,189 @@
+---
+title: "Apache Kafka Broker"
+weight: 30
+type: "docs"
+---
+
+This page shows how to install and configure the 
+[Kafka Broker](https://github.com/knative-sandbox/eventing-kafka-broker).
+
+This guide assumes a 
+[Knative Eventing installation](./../../install/any-kubernetes-cluster.md#installing-the-eventing-component), only
+`eventing-core` is required.
+
+## Installation
+
+```bash
+kubectl apply --filename {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-broker.yaml" >}}
+```
+
+Verify that `kafka-broker-controller`, `kafka-broker-receiver` and `kafka-broker-dispatcher` are running:
+
+```bash
+kubectl get deployments.apps -n knative-eventing 
+
+NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
+eventing-controller            1/1     1            1           10s
+eventing-webhook               1/1     1            1           9s
+kafka-broker-controller        1/1     1            1           3s
+kafka-broker-dispatcher        1/1     1            1           4s
+kafka-broker-receiver          1/1     1            1           5s
+```
```

Comment:
```suggestion
2. Verify that `kafka-broker-controller`, `kafka-broker-receiver` and `kafka-broker-dispatcher` are running, by entering the following command:

```bash
kubectl get deployments.apps -n knative-eventing 
```

Example output:

```
NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
eventing-controller            1/1     1            1           10s
eventing-webhook               1/1     1            1           9s
kafka-broker-controller        1/1     1            1           3s
kafka-broker-dispatcher        1/1     1            1           4s
kafka-broker-receiver          1/1     1            1           5s
```
```

Reply:
idk why when I try to suggest it it's rendering weird, but above is how I'd recommend laying this out (you'd need to indent the output properly too, but as a separate code block to the command).

Reply:
Done: https://github.com/knative/docs/pull/2740/commits/c8969cb556b1dad975a66a0688e7e85d5d985e04